product_docs/docs/eprs/7/installing/index.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: "Installing Replication Server"
+navTitle: Installing
+redirects:
+  - /eprs/latest/03_installation/
+  - /eprs/latest/03_installation/03_installing_rpm_package/
+navigation:
+- linux_x86_64
+- linux_ppc64le
+- windows
+- installing_jdbc_driver
+- installation_details
+- upgrading
+- uninstalling
+---
+
+<div id="installation" class="registered_link"></div>
+
+## [Linux x86-64 (amd64)](linux_x86_64)
+### Red Hat Enterprise Linux (RHEL) and derivatives
+- [RHEL 8](linux_x86_64/eprs_rhel_8), [RHEL 7](linux_x86_64/eprs_rhel_7)
+- [Oracle Linux (OL) 8](linux_x86_64/eprs_rhel_8), [Oracle Linux (OL) 7](linux_x86_64/eprs_rhel_7)
+- [Rocky Linux 8](linux_x86_64/eprs_other_linux_8)
+- [AlmaLinux 8](linux_x86_64/eprs_other_linux_8)
+- [CentOS 7](linux_x86_64/eprs_centos_7)
+### SUSE Linux Enterprise (SLES)
+- [SLES 15](linux_x86_64/eprs_sles_15), [SLES 12](linux_x86_64/eprs_sles_12)
+### Debian and derivatives
+- [Ubuntu 20.04](linux_x86_64/eprs_ubuntu_20), [Ubuntu 18.04](linux_x86_64/eprs_ubuntu_18)
+- [Debian 11](linux_x86_64/eprs_debian_11), [Debian 10](linux_x86_64/eprs_debian_10)
+## [Linux on IBM Power (ppc64le)](linux_ppc64le)
+### Red Hat Enterprise Linux (RHEL) 
+- [RHEL 8](linux_ppc64le/eprs_rhel_8)
+### SUSE Linux Enterprise (SLES) 
+- [SLES 15](linux_ppc64le/eprs_sles_15), [SLES 12](linux_ppc64le/eprs_sles_12)
+## [Windows](windows)
```

Comment:
I think having the section header as the hyperlink to the Windows installer is a bit weird and confusing.  Although it may be somewhat redundant, I think there should be at least a simple sentence following the header with the hyperlink.

Reply:
good point - made some adjustments. I expect we will continue to make some adjustments to this topic as we roll it out and get feedback. For example, we will likely add a stem sentence at the beginning of the topic.